scripts/gulp/plugins/util/getComponentInfo.ts

```diff
@@ -11,7 +11,7 @@ import parseDocblock from './parseDocblock';
 import parseType from './parseType';
 import getShorthandInfo from './getShorthandInfo';
 
-const getAvailableBehaviors = (accessibilityProp: ComponentProp): BehaviorInfo[] => {
+const getAvailableBehaviors = (accessibilityProp: ComponentProp | undefined): BehaviorInfo[] | undefined => {
```

Comment:
```suggestion
const getAvailableBehaviors = (accessibilityProp?: ComponentProp): BehaviorInfo[] | undefined => {
```

Reply:
Nope.

```ts
function a(a: string | undefined) { };
function b(a?: string) { };

console.log(b('a'))
// really optional
console.log(b())

console.log(a('a'))
// requires to pass a param
console.log(a())
```

https://www.typescriptlang.org/play/?ssl=1&ssc=1&pln=11&pc=17#code/GYVwdgxgLglg9mABAQwBTIFyIM5QE4xgDmiAPouACYCmwh1lAlIgN6IC+A3ALABQokWAkQAjdAH4suAsWZsufRbwgJscADbUAdOrhFUYgOTJDjRnwD0FxHmrJ16gJ6I4AByFh7fFWDWadegaoZko+ftq6+mjGpua8VjbUAI4gMLbYiFBwiK7I2BnIOch4yAC23qoaEYFoZkA

Reply:
Alright, makes sense but in this case makes really difference?

Reply:
In this case the param is always passed, so we'll leave it as `ComponentProp | undefined`.

Reply:
Alright :)